quesma/persistence/elastic.go

```diff
@@ -118,11 +118,10 @@ func (p *ElasticJSONDatabase) List() ([]string, error) {
 	}`
 
 	resp, err := p.httpClient.Request(context.Background(), "GET", elasticsearchURL, []byte(query))
-
+	defer resp.Body.Close()
```

Comment:
According to [doc](https://pkg.go.dev/net/http#Client.Do) if `err != nil` body might be `nil`. So it looks like nil dereferencing or do I miss something?